scripts/docker-build

```diff
@@ -9,14 +9,46 @@ mix hex.info
 
 # build cog
 cd /home/operable/cog
+
+# We do this so we know what branch to build for cogctl later on
+COG_BRANCH=`git symbolic-ref HEAD | sed 's!refs/heads/!!'`
```

Comment:
Ah, cool... I'll use that instead.

PS: Obligatory https://git-man-page-generator.lokaltog.net/